src/components/ncm/NodeCoordinatesMatrix.vue

```diff
@@ -60,32 +72,52 @@ export default defineComponent({
 
 <template>
   <div class="wrapper">
-    <table class="table" aria-label="Node Coordinates Matrix">
-      <tbody class="table-body">
-        <tr
+    <div class="table" aria-label="Node Coordinates Matrix">
+      <div class="table-body">
+        <div class="row">
+          <div class="upper-heading-cell">
+            <div class="heading">Type</div>
+          </div>
+          <div ref="upp_heading" class="upper-heading-cell">
+            <div class="heading">Label</div>
+          </div>
+          <div
+            v-for="nodeIndex in matrix[0].coordinates.length"
+            v-bind:key="nodeIndex"
+            class="coordinate-heading"
+          >
+            <div class="heading">{{ nodeIndex }}</div>
+          </div>
+        </div>
+        <div
           v-for="(node, nodeIndex) in matrix"
           v-bind:key="nodeIndex"
           class="row"
         >
-          <th class="heading-cell">
-            <span class="heading">{{ node.heading }}</span>
-            <span v-if="node.subheading" class="subheading">
+          <div ref="headings" class="heading-cell">
+            <div class="heading">{{ node.heading }}</div>
+          </div>
+          <div ref="subheadings" class="heading-cell">
+            <div v-if="node.subheading" class="subheading">
               {{ node.subheading }}
-            </span>
-          </th>
+            </div>
+          </div>
 
-          <td
+          <div
             v-for="(coordinate, coordinateIndex) in node.coordinates"
             v-bind:key="`${nodeIndex}-${coordinateIndex}`"
             class="coordinate-cell"
           >
-            <span class="coordinate-cell-wrapper">
+            <span v-if="coordinate" class="coordinate-cell-wrapper">
```

Comment:
Is it a check for zero? If yes, replace with `coordinate == 0`, because it is more readable.

Reply:
done